pyproject.toml

```diff
@@ -101,6 +101,12 @@ ignore = [
     "S101",
     # Function Arguments
     "PLR0913",
+    # UTC Timezones
+    "DTZ003",
```

Comment:
Working with timezones is a mess, so I think it's better to _always_ use timezone-aware UTC datetimes, that way we can't mess up.

Reply:
Yes sorry i think i forgot to un-commit this